tutorials/step_by_step/singletons_autoload.rst

```diff
@@ -6,48 +6,46 @@ Singletons (AutoLoad)
 Introduction
 ------------
 
-Scene Singletons are very useful things, as they represent a very common
-use case, but it's not clear at the beginning where their value is.
+Scene singletons are very useful, catering to a common use case where you need
+to store persistent information between scenes.
 
-The scene system is very useful, but by itself it has a few drawbacks:
+The scene system by istelf has a few drawbacks:
```

Comment:
Typo here. And I think it was nice to mention that the scene system is useful/great, before saying that it has drawbacks :)